source/github-helpers/index.ts

```diff
@@ -97,8 +97,14 @@ export function getLatestVersionTag(tags: string[]): string {
 }
 
 const escapeRegex = (string: string) => string.replace(/[\\^$.*+?()[\]{}|]/g, '\\$&');
-export const prCommitRegex = new RegExp(`\\b${escapeRegex(location.origin)}[/][^/]+[/][^/]+[/]pull[/]\\d+[/]commits[/][0-9a-f]{7,40}\\b(?! \\]|\\))`, 'gi');
+const prCommitPathnameRegex = /[/][^/]+[/][^/]+[/]pull[/](\d+)[/]commits[/]([\da-f]{7})[\da-f]{33}(?:#[\w-]+)?\b/; // eslint-disable-line unicorn/better-regex
```

Comment:
Why do you use `[/]` instead of `\/`?

Reply:
I generally find them easier to follow. Example, how long does it take to understand what this regex matches?

```js
/^\/\/\*\??/
```

<details>
<summary><code>[] equivalent</code></summary>


```
/^[/][/][*][?]?/
```
matches

```
'//*?' or '//*'
```
</details>